src/lib.rs

```diff
@@ -273,33 +273,52 @@ pub mod auth {
         })
     }
 
+    fn can_log_audit() -> bool {
+        Spi::get_one::<String>("SELECT current_setting('pg_session_jwt.audit_log', true)")
+            .ok()
```

Comment:
Getting the GUC value via SPI is very indirect way to do that. Can't you register a new GUC via extension and then check an in-memory value? At least that's how I'd do it in a standard C extension, I would expect that the same is possible with `pgrx`

Reply:
thanks, would you recommend any specific GUC context? I was thinking about either `sighup` or `postmaster`.